lib/board.rb

```diff
@@ -14,25 +15,20 @@ def width
     @grid.first.length
   end
 
-  def paint!(row, col)
-    @grid[row][col].paint!
-    agent_is_player
+  def paint!(row, col, player)
+    @grid[row][col].paint!(player)
+    @current_player = (player == :player) ? :cpu : :player
```

Comment:
Here's another gameplay decision — players alternate turns. We'd have to refactor to add more players or change that, skip turns, etc.

Reply:
Neat! Since both CPU and player both use the `paint!` method, this automatically swaps `current_player` every time a move is made, in just one line.